### PR TITLE
Pa11y decorated

### PR DESCRIPTION
**Pull request by kkoskelin**
We have had to split our pa11y jobs into several runs -- but we generally want the same default configuration for each run.  To that end, a base config has been created.

Also, a decorator function which can be used to change a simple string URL into a pa11y test object with not only a `url` key but `actions`.  The decorator looks to see that the progress indicator is *not* visible in the page -- because it ought never be present if we are trying to check the accessibility of a successfully-loaded application view.  If it remains on the page through the end of the default timeout (currently 30sec), there is likely a problem somewhere.

Prior to this change, a URL could potentially contain a javascript error which would break the entire application render cycle, but the `pa11y` tool would see nothing amiss in the elements already present in the DOM and would not indicate a failure.

### CODE REVIEW

**Review by matthopson — APPROVED** [review]

**Comment by matthopson** [comment]
I think this seems pretty reasonable so far.

**Review by rachaelparris — APPROVED** [review]

File: web-client/pa11y-ci.base-config.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    decorated = testUrl;
+  }
+  return decorated;
+};
+
+const defaults = {
+  chromeLaunchConfig: {
+    args: ['--no-sandbox'],
+  },
+  concurrency: 1,
```
Comment (matthopson) [inline web-client/pa11y-ci.base-config.js:19]: Definitely gong to impact the run time.

**Review by ericsorenson — APPROVED** [review]

File: web-client/pa11y-ci.base-config.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  debug: true,
+  'include-notices': true,
+  'include-warnings': true,
+  standard: 'WCAG2AA',
+  timeout: 60000,
```
Comment (matthopson) [inline web-client/pa11y-ci.base-config.js:24]: I'm really on the fence about increasing the timeouts. If something under the hood is broken, it's going to wait a full minute before it gives up and moves on, making it take longer to "fail." Is there a real benefit to this?

Comment (ericsorenson) [reply web-client/pa11y-ci.base-config.js:24]: Some of the tests take longer than 30 seconds.

Comment (matthopson) [reply web-client/pa11y-ci.base-config.js:24]: Is that acceptable? Can we reduce that at all?

Comment (matthopson) [reply web-client/pa11y-ci.base-config.js:24]: Perhaps we should create a logger when each URL runs so we _know_ which ones are taking a long time so we can try and address that.

Comment (kkoskelin) [reply web-client/pa11y-ci.base-config.js:24]: What I have come to understand is that this particular `timeout` shown in the configuration relates to the total time necessary to complete the entire test.  Consider when you have a test with a dozen or more actions -- all those actions must complete within `timeout` milliseconds.  However, the 30-second timeout appears to be a per-action setting, something which is internal to the pa11y code and not configurable.
See https://github.com/pa11y/pa11y-ci/issues/61#issuecomment-499374714

File: web-client/pa11y-ci.base-config.js (hunk 1 above)
Comment (kkoskelin) [reply web-client/pa11y-ci.base-config.js:19]: indeed -- although a `concurrency: 1` value is what is currently present on develop (see the deletions from each pa11y config in this PR)

**Pull request merged**